components/ChipTextField/examples/MDCChipTextField.h

```diff
@@ -43,6 +79,6 @@
 
  @param text The string to display in the chip.
  */
-- (void)appendChipWithText:(nonnull NSString *)text NS_SWIFT_NAME(appendChip(text:));
+- (MDCChipView *)appendChipWithText:(nonnull NSString *)text NS_SWIFT_NAME(appendChip(text:));
```

Comment:
So if a client wants an image on the chip (for example because they found an Address Book entry), they are expected to modify the MDCChipView directly after it's appended?

Reply:
The original thought was it should be handled in a delegate method (e.g. shouldAddChipView:chipView or willAddChipView:chipView) if user wants to handle it before the chip is added as a subview.

This code review change was still debatable.

Reply:
The delegate method sounds good. If we can prevent adding a Chip (for example with misspelled contact name) then the client can support a broader range of behavior.

For example:
Types: "rsmoore"
Result: Chip("rsmoore")

Types: "rsmooore"
Result: Text remains set to "rsmooore" and Error is shown

Reply:
Right. That would be supported in this case.